izumi-reflect/izumi-reflect/src/main/scala_3/izumi/reflect/dottyreflection/Inspect.scala

```diff
@@ -33,6 +33,6 @@ object Inspect {
       println(s"$dbs => ${strDbs.size} bytes, ${string2hex(strDbs)}")
       println(strDbs)
     }
-    '{ LightTypeTag.parse(${Expr(ref.hashCode())}, ${Expr(strRef)}, ${Expr(strDbs)}, ${Expr(LightTypeTag.currentBinaryFormatVersion)}) }
+    '{ LightTypeTag.parse(${Expr(ref.hashCode() * 31)}, ${Expr(strRef)}, ${Expr(strDbs)}, ${Expr(LightTypeTag.currentBinaryFormatVersion)}) }
```

Comment:
Should I extract a common function, e.g. `LightTypeTag.hashCodeFromRef`?

Reply:
@Lermex Thanks for taking the time to debug this! 🙏 

Instead of doing `* 31` manually here, we should construct the full LightTypeTag instance in Scala 3 version too, to prevent another desync in the future, e.g. you may change the lines above to

```scala
    val res = {
      val ref = TypeInspections.apply[T]
      val fullDb = TypeInspections.fullDb[T]
      val nameDb = TypeInspections.nameDb[T]
      LightTypeTag(ref, fullDb, nameDb)
    }
```

And make the rest as identical to the Scala 2 version as possible.